packages/vm/src/buildBlock.ts

```diff
@@ -132,11 +132,11 @@ export class BlockBuilder {
    * Calculates and returns the receiptTrie for the block.
    */
   private async receiptTrie() {
-    const gasUsed = new BN(0)
+    let _gasUsed = 0n
```

Comment:
nice, i'd say we don't need the underscore?

Reply:
Do you even use gasUsed there anymore?  It looks like it's an unused variable.

Reply:
ah hm good point, you're right it doesn't look like it's being used! ok it can be removed then :)